helga_jeopardy.py

```diff
@@ -298,13 +556,109 @@ def jeopardy(client, channel, nick, message, cmd, args,
         reset_channel(channel, mongo_db)
         return 'done'
 
+
     # if we have an active question, and args, evaluate the answer
 
     question = mongo_db.find_one({
         'channel': channel,
         'active': True,
     })
 
+    new_game = mongo_db.find_one({
+        'channel': channel,
+        'game_active': True,
+        'game_started': False,
+    })
+
+    current_game = mongo_db.find_one({
+        'channel': channel,
+        'game_active': True,
+        'game_started': True,
+    })   
+
+    if len(args) > 0 and args[0] == 'game':
+        if args[1] == 'new':
```

Comment:
```suggestion
    if args == ['game', 'new']:
```
(but ultimately this should be inside of a `handle_game_command` function or something, as mentioned above).

Reply:
if 3 args were provided and the first 2 were 'game' and 'new', would this still pass?